The CMake install instructions use an incorrect directory for the manpage,
both for the source and destination.
--- ./CMakeLists.txt.orig 2022-02-10 20:58:42 +01:00
+++ ./CMakeLists.txt      2022-03-30 09:21:27 +02:00
@@ -24,4 +24,4 @@
 install(FILES "${CMAKE_BINARY_DIR}/csolrc" DESTINATION /etc/xdg/csol COMPONENT config)
 install(DIRECTORY "${CMAKE_BINARY_DIR}/themes" DESTINATION /etc/xdg/csol COMPONENT config)
 install(DIRECTORY "${CMAKE_BINARY_DIR}/games" DESTINATION /etc/xdg/csol COMPONENT config)
-install(FILES "${CMAKE_BINARY_DIR}/doc/csol.6" DESTINATION ${CMAKE_INSTALL_PREFIX}/man/man6)
+install(FILES "${PROJECT_SOURCE_DIR}/doc/csol.6" DESTINATION ${CMAKE_INSTALL_PREFIX}/share/man/man6)
